dev/com.ibm.websphere.appserver.features/visibility/public/mpTelemetry-1.0/io.openliberty.mpTelemetry-1.0.feature

```diff
@@ -23,7 +24,6 @@ IBM-API-Package: \
   io.openliberty.cdi-3.0, \
   io.openliberty.mpCompatible-5.0,\
   com.ibm.websphere.appserver.injection-2.0, \
-  io.openliberty.grpcClient1.0.internal.ee-9.0,\
   io.openliberty.org.eclipse.microprofile.telemetry-1.0, \
   io.openliberty.grpcClient-1.0,\
   io.openliberty.io.netty,\
```

Comment:
Do you really need this netty dependency? That public feature is still noship so you wouldn't easily be able to move yours to beta with this dependency